agent/src/agent.ts

```diff
@@ -308,4 +310,46 @@ export class Agent extends MessageHandler {
         this.oldClient = client
         return client
     }
+
+    private async logEvent(name: string): Promise<null> {
```

Comment:
@tjdevries - for Event Logging Everywhere, we're working on a new SDK for recording events using a new mutation and event schema, so that individual instances report telemetry. There's a draft PR here: https://github.com/sourcegraph/cody/pull/1192

This won't really be available until late next week at the earliest, but it would be nice to have this new agent protocol reflect the API we plan to introduce, e.g. by changing this a bit here:

```suggestion
    private async recordEvent(feature: string, action: string): Promise<null> {
```

cc @nathan-downs

Reply:
A bit more context here: https://docs.sourcegraph.com/dev/background-information/telemetry

I will be OOO this week but can cc @nathan-downs @valerybugakov (who we just shared some context with today) for discussion in Slack if you have any questions!

Reply:
What I see there mostly makes sense, I think we can continue to push this PR forward and then replace the implementation with the v2 stuff once it lands. Thanks for the heads up